docs/docs/cmd/spo/group/group-member-add.mdx

```diff
@@ -33,18 +33,24 @@ m365 spo group member add [options]
 `--userIds [userIds]`
 : The user Id of the user to add as a member. (Id of the site user, for example: 14) If multiple users need to be added, the Ids have to be comma-separated. Specify either `userIds`, `userNames`, `emails`, `aadGroupIds` or `aadGroupNames`.
 
+`--entraGroupIds [entraGroupIds]`
+: The object Id of the Entra group to add as a member. If multiple groups need to be added, the Ids have to be comma-separated. Specify either `userIds`, `userNames`, `emails`, `aadGroupIds`, `entraGroupIds`, `aadGroupNames`, or `entraGroupNames`.
```

Comment:
@SmitaNachan I actually rechecked this with the team and we decided that the docs should reflect what is in the code. As it will actually work that way the user may specify either new or old options what you wrote here is correct 👍👍
Sorry for the confusion and I will resolve the comments that are not anymore valid 🙏